content/docs/contributing/core.md

```diff
@@ -142,173 +143,6 @@ To pass additional arguments:
 $ python -m tests --pdb
 ```
 
-## Testing remotes
```

Comment:
I have removed everything of this because now they reside in individual plugins. Writing a guide here for that is out of the scope.

We also don't need credentials for many of the plugins, and should work out of the box (except `oss` and `gdrive` IIRC).